topo/node/srl/srl_test.go

```diff
@@ -310,3 +310,100 @@ func TestResetCfg(t *testing.T) {
 		})
 	}
 }
+
+func TestConfigPush(t *testing.T) {
+	ki := fake.NewSimpleClientset(&corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "pod1",
+		},
+	})
+
+	reaction := func(action ktest.Action) (handled bool, ret watch.Interface, err error) {
```

Comment:
is the watch reactor required here?

Reply:
yes, you're right. I have removed it from the two tests that don't require it.